db.go

```
@@ -0,0 +1,65 @@
+import "database/sql"
+
+var db *sql.DB
+
+// todo check how headscale do SQL db init
+func initDatabase(dataSrcName string) (*sql.DB, error) {
+	db, err := sql.Open("sqlite3", dataSrcName)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = db.Exec("CREATE TABLE IF NOT EXISTS running_jobs (key VARCHAR(50) PRIMARY KEY, retry_count INTEGER, merge_request_id INTEGER)")
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)